src/components/fields/TraceSelector.js

```diff
@@ -120,18 +128,36 @@ class TraceSelector extends Component {
       return (
         <div>
           <Field {...props}>
-            <TraceTypeSelectorButton
-              {...props}
-              traceTypesConfig={this.context.traceTypesConfig}
-              handleClick={() =>
-                this.context.openModal(TraceTypeSelector, {
-                  ...props,
-                  glByDefault: this.context.glByDefault,
-                })
-              }
-            />
+            <div
+              style={{
+                display: 'flex',
```

Comment:
could we put this into a stylesheet :)

Reply:
this style will not be reused anywhere, the main purpose of it is to override `field__widget`'s flex-wrap style